lib/util/hud_lists.rb

```diff
@@ -57,7 +57,7 @@ def record_type_map
       162 => 'RHY referral',
       200 => 'Bed night',
       210 => 'HUD-VASH OTH voucher tracking',
-      300 => 'C2 Moving On Assistance Provided',
+      300 => 'Moving On Assistance',
```

Comment:
@eanders @awisema this affects warehouse code, is it OK?
The string was incorrect before: https://hudhdx.info/Resources/Vendors/HMIS_CSV_Specifications_FY2022_v1.3.pdf#page=42

Reply:
Yes, thank you!